bytelatent/transformer.py

```diff
@@ -94,16 +78,25 @@ def forward(
         target: Optional[torch.Tensor] = None,
         tok_idx: Optional[torch.Tensor] = None,
         mask: Optional[Union[BlockMask, AttentionBias, torch.Tensor, str]] = None,
-        attn_impl: str = "sdpa",
+        attn_impl: str | None = None,
     ):
+        if attn_impl is None:
```

Comment:
It might be clearer in a single line:
attn_impl = attn_impl or self.attn_impl

Reply:
I think I actually prefer the explicit style, but not super strong preference.